scripts/setup-app.sh

```diff
@@ -0,0 +1,52 @@
+if [ -z "$CHURCH" ] || [ -z "$APOLLOS_API_KEY" ]; then
+    echo "CHURCH and APOLLOS_API_KEY required!"
+    exit 1
+fi
+
+API_URL="https://apollos-cluster-production.herokuapp.com/api"
+APP=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP_NAME")
+IOS_BUNDLE=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.APPLE_BUNDLE_ID")
+ANDROID_BUNDLE=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP_ANDROID_PKG_ID")
+SERVER_URL=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP_SERVER_URL")
```

Comment:
```suggestion
SERVER_URL=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.SERVER_URL")
```

maybe in the future we could come up with a way to pull this from the church_routes table :)

Reply:
Yea I wanted to. but the fact that we have a routes table prevents that :)